sound/soc/intel/boards/Kconfig

```diff
@@ -606,25 +623,6 @@ config SND_SOC_INTEL_SOF_CML_RT1011_RT5682_MACH
 
 endif ## SND_SOC_SOF_COMETLAKE && SND_SOC_SOF_HDA_LINK
 
-if SND_SOC_SOF_JASPERLAKE
-
```

Comment:
it would be simpler if you just removed the if JASPERLAKE and updated the select/depends below. You changed the order so it's a bit hard to review really.

Reply:
ok. I'll move it back.